manifests/init.pp

```diff
@@ -66,7 +69,8 @@
   String $postgresql_db_password = extlib::cache_data('pulpcore_cache_data', 'db_password', extlib::random_password(32)),
   String $django_secret_key = extlib::cache_data('pulpcore_cache_data', 'secret_key', extlib::random_password(32)),
   Integer[0] $redis_db = 8,
-  Stdlib::Fqdn $servername = $facts['fqdn']
+  Stdlib::Fqdn $servername = $facts['fqdn'],
+  String $django_remote_user = 'HTTP_REMOTE_USER'
```

Comment:
Could you ensure a trailing comma?
```suggestion
  String $django_remote_user = 'HTTP_REMOTE_USER',
```

I'm also wondering if we should make this optional:
```suggestion
  Optional[String] $django_remote_user = undef,
```

That way we are secure by default, unless the vhost has been set up to properly set it in a secure fashion.

Reply:
Hey @ekohl ,

Thanks for the feedback. I added the trailing comma. Regarding the second suggestion, if this parameter should undefined by default, where do we set it so that it's present where Katello needs it?

Reply:
https://github.com/theforeman/puppet-foreman_proxy_content/blob/cf4cc2f6b402c447173e108d807ef28c0e2d75f9/manifests/init.pp#L295-L309
The first part is where this class is called and the `foreman::config::apache::fragment` actually sets the header.

Reply:
If the parameter is optional and undefined by default here, then I think we can assign some value to it when we declare the class here: https://github.com/theforeman/puppet-foreman_proxy_content/blob/master/manifests/init.pp#L295-L298

We would also then make the line of the template `settings.py.erb` optional by placing it inside an erb `if` block.

@ekohl is this the type of approach you have in mind?

Reply:
@wbclark exactly